### DISCUSSION TITLE
Possible to cut a new release/help with it?

### DISCUSSION THREAD

**Discussion by futile** [wezterm/wezterm#4561] [General]
Hi,

I wanted to ask if it is possible to cut a new wezterm-release soon~ish?

Reading other issues/discussions I have gathered that it's "simply" difficult due to time-issues (thanks for the project, wezterm is amazing!). The current version is, given wezterm's rate of change, a bit behind, and there is a bugfix I would like to have in a release 😇

Since it's (also) an issue of time, I wanted to ask if it's possible to help/assist with the release process?

Thanks :)

**Comment by wez** [comment]
Thanks for the offer to help; actually making the release isn't too much effort; it's essentially just pushing a tag and then doing a search/replace on some docs.

What's stopping me from cutting a release today is this issue, which is an obvious regression:
* https://github.com/wez/wezterm/issues/4231

I just haven't had the time or motivation to sit down and fix it.

**Reply by hyperpuncher** [reply]
I think this fixes the issue, tested on linux and mac with different font sizes.
* #4752

**Comment by futile** [comment]
Thank you for the update! Was a bit busy with work, but wanted to take a look at the issue you linked now. Just saw it's a Windows-issue though, and my Windows setup is... subpar at the moment :see_no_evil: But I'll ask for some info in the issue (since I'm not familiar enough with wezterm/terminal emulators to understand the problem as-is), and then see from there :+1:

**Comment by damonto** [comment]
@wez Could you please merge this PR #3699 before cutting a new release? I really need it.
